diff --git a/target/product/full_base.mk b/target/product/full_base.mk
index 9ad47b24b..c8e0f6c83 100644
--- a/target/product/full_base.mk
+++ b/target/product/full_base.mk
@@ -43,7 +43,7 @@ PRODUCT_PROPERTY_OVERRIDES := \
     ro.config.notification_sound=pixiedust.ogg
 
 # Put en_US first in the list, so make it default.
-PRODUCT_LOCALES := en_US
+PRODUCT_LOCALES := zh_CN
 
 # Get some sounds
 $(call inherit-product-if-exists, frameworks/base/data/sounds/AllAudio.mk)
